typings/index.d.ts

```diff
@@ -3195,6 +3202,8 @@ declare module 'discord.js' {
     reason?: string;
   }
 
+  type WebhookEditMessageOptions = Pick<WebhookMessageOptions, 'embeds' | 'allowedMentions'>;
```

Comment:
This feels wrong... You can still patch the content...am I missing something?

Reply:
`content` is the [2nd param](https://github.com/discordjs/discord.js/pull/5223/files#diff-b764df7a2f9c6fb1b9c444c4d67decafb4bc3df8bf7310ccd70b245b4e478d60R229) and isn't a part of the options. It would have been nicer if it was within the `options` object but I decided to go with this approach to be consistent with the other send/edit methods.

Reply:
Will your current typings support `{ content: 'abalabahaha', embeds: [new MessageEmbed().setDescription('owo')] }`?

Reply:
It does not, it would have to be passed as:
```js
webhook.editMessage(message, 'abalabahaha', { embeds: [new MessageEmbed().setDescription('owo')] })
```
Which would be consistent with how d.js handles other send/edit methods.

Reply:
![image](https://user-images.githubusercontent.com/17960496/105529217-bdba3100-5cee-11eb-9955-3e255130ce5a.png)
![image](https://user-images.githubusercontent.com/17960496/105529231-c3177b80-5cee-11eb-94a8-169d2b1c8866.png)

Seems like our types are really, really wack.

Reply:
Looks like that's unintentionally being done over here: https://github.com/discordjs/discord.js/blob/f95f5dcd791b39c6a4d60dc8d64b0287e06ed768/src/structures/APIMessage.js#L84-L86
So that editing the messge without setting the content doesn't mess with `Message#content` (1673b6f8f5bc53a30e2f2ef1123057d4e50c37c8)... buttdoesn't seem to be documented on any of the JSDoc typedefs nor the typings, I guess that would be out of the scope of this PR to fix that so feel free to open another PR for this or I don't mind doing that sometime later.

Reply:
Resolved with cca8632370c2eba84b9c50bddb7e54d698eaf9f0, one thing to note is that this will also allow `send('foo', { content: 'bar' })` since this technically works due to how the `APIMessage` will resolve the content.